docs/spec/supervisor/supervisor.md

```diff
@@ -0,0 +1,153 @@
+
+```rust
+/// Stores the Light Blocks which could not be confirmed to have originated from
+/// the same chain as the predecesor light blocks.
+type FetchedStack = Stack<LightBlock>;
```

Comment:
What are the semantics of the stack? I don't believe there is a type definition in this file.

Reply:
There is no semantics as we could use a `Vec` for this, the goal was just to name it here and explain it is going to be used as a stack (push, pop, peek, is_empty) and not with random access.